lib/developer_portal/app/views/developer_portal/admin/account/authorize_net/credit_card_error.html.erb

```diff
@@ -1,9 +0,0 @@
-Could not retrieve credit card info. Possible reasons:
-<ul>
-  <li>You don't
-  have <a href="http://www.authorize.net/solutions/merchantsolutions/merchantservices/cim/">Authorize.net
-  CIM enabled</a> . Make sure it is selected in your authorize.net
-  control panel.</li>
- <li>Merchant credentials are incorrect. Please double check your credentials are ok.</li>
- <li>Payment gateway is down.</li>
-</ul>
```

Comment:
Is this not required anymore? it seems to be not only exclusive to Authorize.net

Reply:
Used only by authorized net:
```
git grep credit_card_error
lib/developer_portal/app/controllers/developer_portal/admin/account/authorize_net_controller.rb:      render action: 'credit_card_error'
```